portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,70 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Ibiyemi's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+  <base target="_blank" />
+</head>
+
+<body>
+  <header>
+    <h1>
+      Ibiyemi Abiodun
+    </h1>
+  </header>
+  <section id="content">
+    <h2>
+      Hi, I'm Ibiyemi Abiodun!
+    </h2>
+    <p>
+      I'm a rising sophomore at Cornell, double majoring in electrical & computer engineering, and
+      computer science. I love cooking & baking, karate, VFX, cinematography, and of course, coding.
+    </p>
+    <p>
+      I was born in Nigeria, and I've lived all over the U.S., although when people ask I tell them
+      I'm from Seattle because I lived in Bellevue & Kirkland for about 9 years of my 19-year life.
+    </p>
+    <h2>Career</h2>
+    <p>
+      You want my resume? <a href="/dl/resume.pdf">Here you go.</a>
+    </p>
+    <h2>Cooking</h2>
+    <p>
+      I know, we're in quarantine, everyone is cooking and baking. But look at this:
+    </p>
+    <div class='cooking-dish'>
+      <img src="/img/orange-chicken.jpg" alt="Spicy orange chicken with butter rice" />
+      <h3>Spicy orange chicken with butter rice</h3>
+      <p>
+        This is based on
+        <a href="https://www.youtube.com/watch?v=N_87Ow7HTdQ">Joshua Weissman's recipe</a>. I don't
+        have Serrano chiles, because my mom doesn't believe in them, but I was able to compensate by
+        adding just a little bit of habañero oil.
+      </p>
     </div>
-  </body>
+    <div class='cooking-dish'>
+      <img src="/img/cornbread.jpg" alt="Maple syrup cornbread with pork sausage and barbecue sauce"
+      />
+      <h3>Maple syrup cornbread with pork sausage and barbecue sauce
+      </h3>
+    </div>
+    <div class='cooking-dish'>
+      <img src="/img/raspberry-cheesecake.jpg" alt="Raspberry swirl cheesecake" />
+      <h3>Raspberry swirl cheesecake</h3>
+    </div>
+    <div class='cooking-dish'>
+      <img src="/img/raspberry-a-la-mode.jpg" alt="Vanilla ice cream with raspberry reduction" />
+      <h3>Vanilla ice cream with raspberry reduction</h3>
+    </div>
+  </section>
+  <footer>
+    <p>&copy; 2020 Google, Inc.</p>
```

Comment:
I think you should be able to directly use the © symbol here. In general you want to avoid using entity references.

Style guide: https://google.github.io/styleguide/htmlcssguide.html#Entity_References